homeassistant/components/isy994/entity.py

```diff
@@ -75,7 +75,7 @@ def async_on_control(self, event: NodeProperty) -> None:
             # New state attributes may be available, update the state.
             self.async_write_ha_state()
 
-        self.hass.bus.fire("isy994_control", event_data)
+        self.hass.bus.async_fire("isy994_control", event_data)
```

Comment:
Side note: Entities shouldn't fire custom events.

Reply:
@bdraco - as we've discussed before, long term path forward for these legacy events is to get migrated to Device Actions, I just haven't had time to tackle that yet.